src/files.js

```diff
@@ -492,5 +492,9 @@ FOAM_FILES([
   { name: "foam/crypto/hash/Hashable" },
   { name: "foam/crypto/sign/Signer" },
   { name: "foam/crypto/sign/Signable" },
-  { name: "foam/crypto/sign/SignedFObject" }
+  { name: "foam/crypto/sign/SignedFObject" },
+
+//help desk support 
+  { name: "foam/support/model/SupportEmail" }
```

Comment:
Move to own support.js file (like nanos.js) rather than including in foam core which will make all foam apps larger, even those that don't use the support feature.